var/spack/repos/builtin/packages/xforms/package.py

```diff
@@ -20,6 +20,7 @@ class Xforms(AutotoolsPackage):
     depends_on("libx11", type="link")
     depends_on("libxpm", type="link")
     depends_on("jpeg", type="link")
+    patch("xformsPatch.patch")
```

Comment:
Do you know if the patch applies to all listed versions or for some version on?

See the docs at https://spack.readthedocs.io/en/latest/packaging_guide.html#patches.

Reply:
That patch is needed for the last version of xforms, 1.2.4 . Not sure about the other versions though.

Reply:
Do you want to add the `when="@1.2.4"` argument to the patch then?